web/src/components/query-summary/__tests__/query-summary.spec.tsx

```diff
@@ -20,7 +20,7 @@ describe('<QuerySummary />', () => {
   it('should show summary', async () => {
     const wrapper = mount(<QuerySummary {...mocks} />);
     expect(wrapper.find('#flowsCount').last().text()).toBe('3 flows');
-    expect(wrapper.find('#bytesCount').last().text()).toBe('161400 bytes');
+    expect(wrapper.find('#bytesCount').last().text()).toBe('161 kB');
```

Comment:
nitpick. Shouldn't be `K` in uppercase?

Reply:
kB is an exception in SI standard: https://en.wikipedia.org/wiki/Kilobyte
there's MB, GB, TB ... but kB :)